SocialNetwork/SocialNetwork.Core/Interfaces/IUserService.cs

```diff
@@ -1,15 +1,13 @@
 ﻿using SocialNetwork.Core.Models;
-
 namespace SocialNetwork.Core.Interfaces;
-
-// Single Responsibility
-
 public interface IUserService
 {
-    // pagination
-    IEnumerable<User> GetAllUsers(int skip = 0, int take = 20);
-
-    Task<User> AddUser(User user);
-    Task<User> UpdateUser(User user);
-    Task DeleteUser(int id);
+    List<User> Users { get; set; }
```

Comment:
Remove this. We have method GetUsers() ti access users. Having a public list of users can cause incorrect access to data